CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
```

Comment:
Are you using prettier? There should be a line break between those two titles.

Reply:
We decided to keep the changelogs like this in our apps, but we can discuss this further.

Reply:
@kevinch It's something @klzns showed at some FEG meeting a while ago. Sometimes that empty line can lead to a annoying conflict :l